apps/files/src/default.js

```diff
@@ -33,7 +33,10 @@ const appInfo = {
       app: 'file-link',
       component: FileLink,
       enabled (capabilities) {
-        return capabilities.files_sharing.public.enabled === '1'
+        if (capabilities.files_sharing) {
+          return capabilities.files_sharing.public.enabled === '1'
```

Comment:
This won’t work as soon as the owncloud-sdk is updated. Value was changed to true/false. See https://github.com/owncloud/phoenix/pull/2099#discussion_r329623409